src/components/anchors.js

```diff
@@ -65,7 +48,9 @@ export default class Anchors extends React.PureComponent {
               }`}
               style={{ color: isSelected ? color : null }}
             >
-              <div dangerouslySetInnerHTML={{ __html: this.getSVG(id) }} />
+              <div className="emoji-mart-anchor-icon">
+                {iconFns.categories[id]()}
```

Comment:
Why does this need to be a function ?

Reply:
We decided to make this a function so that the icons can be React components, in order to give the user some flexibility with what they want to use for the icons, whether it be PNG or SVG

Reply:
Oh I see now, overlooked that part I was looking at the SVG data but missed the back ticks `